README.md

```diff
@@ -169,6 +171,39 @@ For example:
               value: "21:0:0"
 ```
 
+#### Set scheduler
+`SCHEDULER_CRON_SECONDS`, `SCHEDULER_CRON_MINUTES`, `SCHEDULER_CRON_HOURS`, `SCHEDULER_CRON_DAY_OF_MONTH`, `SCHEDULER_CRON_MONTH`, `SCHEDULER_CRON_DAY_OF_WEEK`, and `SCHEDULER_CRON_YEARS` can be used to specify the scheduler in which updates are permitted.
+When `SCHEDULER_CRON_SECONDS`, `SCHEDULER_CRON_MINUTES`, `SCHEDULER_CRON_HOURS`, `SCHEDULER_CRON_DAY_OF_MONTH`, `SCHEDULER_CRON_MONTH`, `SCHEDULER_CRON_DAY_OF_WEEK`, and `SCHEDULER_CRON_YEARS` are both * (default), the feature is disabled.
+
+To enable this feature, go to `bottlerocket-update-operator.yaml`, change `SCHEDULER_CRON_SECONDS`, `SCHEDULER_CRON_MINUTES`, `SCHEDULER_CRON_HOURS`, `SCHEDULER_CRON_DAY_OF_MONTH`, `SCHEDULER_CRON_MONTH`, `SCHEDULER_CRON_DAY_OF_WEEK`, and `SCHEDULER_CRON_YEARS` to a cron expression allowed formatted value (UTC (24-hour time notation)).
+
+Note: brupop uses UTC (24-hour time notation), please convert your local time to UTC.
```

Comment:
I'm fine if this doesn't get changed, but we state `UTC (24-hour time notation)` a few places now, which makes it sound like it is just a date format.

I would hope most people are aware of UTC, but it would be good the first time to expand it out to the full "Coordinated Universal Time" instead of just the acronym, and rather than saying "24-hour time notation" each time, provide a link to something like [a time converter](https://www.timeanddate.com/worldclock/converter.html?iso=20230124T060000&p1=1440) to help the user figure out the UTC time for their desired window.